code/modules/mob/living/combat/parry.dm

```diff
@@ -321,6 +342,13 @@
 /mob/proc/do_unarmed_parry(parrydrain as num, mob/living/user)
 	if(ishuman(src))
 		var/mob/living/carbon/human/H = src
+		if(istype(user, /mob/living/simple_animal/hostile/rogue/jitterskull))
```

Comment:
Again, putting something hyperspecific edgecases like this into parry code probably isn't the best practice for this.
As above, if you want to make them be able to deal with parries, give the mobs unarmed skill.